src/main/java/com/es/jointexpensetracker/web/CommonExpenseServlet.java

```diff
@@ -0,0 +1,61 @@
+package com.es.jointexpensetracker.web;
+
+import com.es.jointexpensetracker.model.Expense;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Currency;
+
+public class CommonExpenseServlet extends HttpServlet {
+
+    protected boolean changeExpenseByRequest(Expense editExpense, HttpServletRequest request){
```

Comment:
bad method name.
There is a common acronym: CRUD which stands for:
Create
Read
Update
Delete
Try to use this words.
So rename the method to updateExpenseFromRequest